### DISCUSSION TITLE
Modbus RTU

### DISCUSSION THREAD

**Discussion by TomasAdvance** [frangoteam/FUXA#429] [General]
Hi May I ask you for what shall be setup in serial port settings in Modbus RTU.
![16584855705117843785531844127159](https://user-images.githubusercontent.com/97446916/180420214-1e8da997-aa08-4ee4-9704-8c81ad6ec519.jpg)
I try to write down several options but without out success.
I know the device are available in com4, Ai have test it by modbus pool.
Thank for your support

**Comment by unocelli** [comment]
Hi,
try to install your self the modbus-serial by server: `npm i modbus-serial` , and serial port without ()
